vispy/visuals/colorbar.py

```diff
@@ -598,7 +598,7 @@ def label(self, label):
 
     @property
     def label_str(self):
-        return self._label_str
```

Comment:
@macrocosme This will not work. There is no self._colorbar in vispy/visuals/colorbar.py

I meant to just add some docstring here.

Reply:
Please revert this change.

Reply:
I'm at a conference. I'll get back to it when I have a bit more time to concentrate! : )
This should be soon.